packages/angular/ssr/src/manifest.ts

```diff
@@ -123,8 +147,11 @@ let angularAppManifest: AngularAppManifest | undefined;
  *
  * @param manifest - The manifest object to set for the Angular application.
  */
-export function setAngularAppManifest(manifest: AngularAppManifest): void {
-  angularAppManifest = manifest;
+export function setAngularAppManifest(manifest: AngularAppManifestConfig): void {
+  angularAppManifest = {
+    ...manifest,
+    assets: new Map(Object.entries(manifest.assets)),
```

Comment:
@dgp1130, I’m wondering if using a `Map` here is excessive, and if we should just stick with a plain `Record`. I understand the decision to avoid prototype pollution, but it feels like an overly cautious approach.

We are only using the this to read data, with no need for iteration or modification. Given this, I believe a `Record` is a better choice, since there's no need for the features that a `Map` offers, such as key ordering or methods like `set` or `delete`.

Reply:
`Object.create(null)` creates a object without prototype, if that's the only concern, right?

Reply:
I'm ok with a `Record`. My suggestion is more to prefer `Map` where possible, but when there are compelling reasons to keep the `Record` (like inlining in a manifest), I think it's perfectly reasonable to consider.